web/src/installers/index.ts

```diff
@@ -53,6 +54,7 @@ export const kubernetesConfigSchema = {
     useStandardNodePortRange: { type: "boolean" },
     controlPlane: { type: "boolean", flag: "control-plane", description: "Used during a join script to indicate that the node will be an additional master (read-only)" },
     certKey: { type: "string", flag: "cert-key", description: "A secret needed for new master nodes to join an existing cluster (read-only)" },
+    cisCompliance: { type: "boolean", flag: "kubernetes-cis-compliance", description: "Enables CIS compliance" },
```

Comment:
just like you added this, you need to remove the doc for the kurl.cisCompliance option